sys/include/fido2/ctap/ctap_mem.h

```diff
@@ -155,6 +142,20 @@ int fido2_ctap_mem_get_flashpage_number_of_rk(uint16_t rk_idx);
  */
 int fido2_ctap_mem_get_offset_of_rk_into_flashpage(uint16_t rk_idx);
 
+/**
+ * @brief Get page number for storing authenticator state information
+ *
+ * @return page number
+ */
+unsigned fido2_ctap_mem_get_state_flashpage_number(void);
+
+/**
+ * @brief Get start page for storing resident keys
+ *
+ * @return page number
+ */
+unsigned fido2_ctap_mem_get_rk_start_page(void);
+
```

Comment:
How about `get_state_flashpage` and `get_rk_flashpage` to be a bit more consistent?